### DISCUSSION TITLE
Volume slider for videos

### DISCUSSION THREAD

**Discussion by SkyLord52** [Docile-Alligator/Infinity-For-Reddit#916] [Ideas]
Add you a volume slider for videos, instead of it just being based on your phone's volume. That way I can watch videos with no sound while still being able to listen to music.

**Comment by Docile-Alligator** [comment]
OK, I will try.

**Comment by slackerbob** [comment]
Also a persistent mute button. I rarely want to hear Reddit videos, so I usually have them muted by default on other Reddit apps.

**Comment by Docile-Alligator** [comment]
Good idea

**Comment by slackerbob** [comment]
Thank you for adding the mute button! It's been very useful!

**Comment by Docile-Alligator** [comment]
You are very welcome.
